packages/hierarchies/src/hierarchies/HierarchyFiltering.ts

```diff
@@ -334,19 +344,17 @@ export function createHierarchyFilteringHelper(
         reducer.accept(normalizedPath);
       }
       if (matchedPathPromises.length === 0) {
-        return reducer.getNodeProps();
+        return reducer.getNodeProps(props.parentNode);
       }
       return Promise.all(matchedPathPromises)
         .then((matchedPath) => matchedPath.forEach((normalizedPath) => normalizedPath && reducer.accept(normalizedPath)))
-        .then(() => reducer.getNodeProps());
+        .then(() => reducer.getNodeProps(props.parentNode));
     },
   };
 }
 
 /** @public */
-export type NodeProps = Pick<HierarchyNode, "autoExpand" | "filtering"> & {
-  filtering?: { autoExpandDepthInPath?: number; autoExpandDepthInHierarchy?: number };
-};
+export type NodeProps = Pick<HierarchyNode, "autoExpand" | "filtering">;
```

Comment:
Do we need this type anymore? I think `getNodeProps` could just return `Pick<HierarchyNode, "autoExpand" | "filtering">`. If not, could we change its name to something more appropriate?.. It shows up in our API markdown and is way too generic to understand what it is.